### DISCUSSION TITLE
The meaning of train/box_om and train/box_oo?

### DISCUSSION THREAD

**Discussion by Naaimaben** [THU-MIG/yolov10#217] [Q&A]
Hello,

Thank you and the team for the new release of YOLOV10!

I have a question about the new validation charts in YOLOV10. Could you please explain the meaning of the new charts, such as train/box_om and train/box_oo?

Thank you for your time.

![WhatsApp Image 2024-06-07 at 15 43 02](https://github.com/THU-MIG/yolov10/assets/40637137/6bba52df-60e0-481a-8d0b-125c834d690b)

**Comment by MengmingWu** [comment]
I guess om refers to one to many in the detection head, and oo is one to one
